swap/protocol_test.go

```diff
@@ -296,3 +303,156 @@ func TestTriggerDisconnectThreshold(t *testing.T) {
 		t.Fatalf("Expected still no cheque, but there are %d", lenCheques)
 	}
 }
+
+// TestSwapRPC tests some basic things over RPC
+// We want this so that we can check the API works
+func TestSwapRPC(t *testing.T) {
+
+	var (
+		p2pPort = 30100
```

Comment:
It would be good to discover a free port to listen on. I think that there is a function in the codebase to do so in almost reliable way, but good enough, maybe assignTCPPort in cmd/swarm/config_test.go. But I am not sure if it is needed as rpc is connected over ipc, it works only with 0 value.

Reply:
Indeed, I removed the port altogether, works fine as it is only ipc